src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -124,10 +124,22 @@ export const AccountTransactions: React.FC = () => {
           entry.description ?? '',
           entry.category?.name ?? entry.category?.code ?? '',
           entry.type === FinancialAccountEntryTypeEnum.Debit
-            ? currencyFormat(formatNumber(entry.amount), entry.currency, locale)
+            ? currencyFormat(
+                formatNumber(
+                  entry.amount.replace(/[\xA0\u2000-\u200B\uFEFF]/g, ''),
```

Comment:
For some reason this didn't work. There are still strange characters in the export.

Reply:
I think that the `replace` needs to be on what `currencyFormat` returns because `currencyFormat` is the function creating the weird characters.

Reply:
Sorry, do you think we should replace the weird spaces with regular spaces instead of removing them? I think "RUB 100" would be better than "RUB100" in the export.